crates/matrix-sdk-base/src/store/memory_store.rs

```diff
@@ -243,6 +243,39 @@ impl MemoryStore {
 
         for (room, events) in &changes.stripped_members {
             for event in events.values() {
+                match event.content.membership {
+                    MembershipState::Join => {
+                        self.joined_user_ids
```

Comment:
Isn't this still mixing stripped user ids and non-stripped ones?

Reply:
sorry, you are right.